### PR TITLE
ci: update dependabot configuration

### PR DESCRIPTION
**Pull request by maliroteh-sf**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -3,12 +3,14 @@ updates:
     - package-ecosystem: 'npm'
       directory: '/'
       schedule:
-          interval: 'weekly'
-          day: 'saturday'
+          interval: 'monthly'
+          day: 'monday'
```
Comment (khawkins) [inline .github/dependabot.yml:7]: According to [the docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval), the `monthly` option always happens on the first of the month, is not configurable.

Comment (khawkins) [reply .github/dependabot.yml:7]: (It's not 100% clear on time/timezone application, so I'd suggest leaving those in.)

**Review by khawkins — APPROVED** [review]
Other than taking out the superfluous `day` configuration, this looks fine to me. 👍

File: .github/dependabot.yml (hunk 1 above)
Comment (maliroteh-sf) [reply .github/dependabot.yml:7]: But the first day of the month may not be a Monday so I thought to explicitly pick the day of the week.

Comment (khawkins) [reply .github/dependabot.yml:7]: At least from the docs, it appears that it's not configurable. We can try it and see I suppose. We'll know for sure either on 9/1 or 9/2, whether it worked.

**Pull request merged**